packages/flutter/test/cupertino/scrollbar_test.dart

```diff
@@ -961,7 +961,13 @@ void main() {
 
     await tester.pumpAndSettle();
 
-    final dynamic exception = tester.takeException();
+    dynamic exception = tester.takeException();
```

Comment:
While you're here, can we get rid of `dynamic`?  Perhaps even just changing this to `AssertionError? exception = tester.takeException() as AssertionError?`.